mapviz_plugins/src/textured_marker_plugin.cpp

```diff
@@ -95,9 +100,32 @@ namespace mapviz_plugins
 
   void TexturedMarkerPlugin::ClearHistory()
   {
+    ROS_INFO("Marker Clear all");
```

Comment:
This is just personal preference, but when I have a log statement that is only used to show every time a function is called, I prefer for it to print out the name of the function; e. g.,
```cpp
ROS_INFO("TexturedMarkerPlugin::ClearHistory()");
```
That helps to disambiguate it from any other logging statement that might be related to clearing markers, and if somebody else later sees this in the output and is trying to track down where it came from, it makes it more obvious.

Reply:
I agree this should be a more specific message.  However, note that the text here is currently also displayed on the GUI under the 'Status' tagline.

Reply:
Are you sure?  I would only expect that if you're calling `PrintInfo`; `ROS_INFO` shouldn't change anything in the GUI.

Reply:
Seems like it does not anymore.  Must have only happened in testing.